### PR TITLE
Ut 2995 export anim curve tangents

### PR DESCRIPTION
**Pull request by vkovec**

### CODE REVIEW

File: com.unity.formats.fbx/Editor/FbxExporter.cs [hunk 1]
```diff
@@ -1745,23 +1752,9 @@ internal static HashSet<float> GetKeyTimes(AnimationCurve[] animCurves)
         /// NOTE : This is a work in progress (WIP). We only export the key time and value on
         /// a Cubic curve using the default tangents.
         /// </summary>
-        internal void ExportAnimationKeys (AnimationCurve uniAnimCurve, FbxAnimCurve fbxAnimCurve, 
-            UnityToMayaConvertSceneHelper convertSceneHelper)
+        internal static void ExportAnimationKeys (AnimationCurve uniAnimCurve, FbxAnimCurve fbxAnimCurve, 
+            UnityToMayaConvertSceneHelper convertSceneHelper, string uniPropertyName)
```
Comment (BadSingleton) [inline com.unity.formats.fbx/Editor/FbxExporter.cs]: is it me or the uniPopertyName parameter is not being used anywhere?

Comment (vkovec) [reply com.unity.formats.fbx/Editor/FbxExporter.cs]: you're right I don't use it anymore, good catch

**Comment by benoithudson** [comment]
Seems like the same set of failures as the other PR.

**Comment by vkovec** [comment]
I think it's failing because I didn't update the com.autodesk.fbx dependency

**Pull request merged**